Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -372,6 +372,9 @@ - (void)_initializeWithConfiguration:(WKWebViewConfiguration *)configuration
     _allowsLinkPreview = linkedOnOrAfterSDKWithBehavior(SDKAlignedBehavior::LinkPreviewEnabledByDefault);
     _findInteractionEnabled = NO;
 
+    _pendingFindLayerID = 0;
```

Comment:
Do we also need to zero these out on a web process crash?

Reply:
Good point, I did this in `-[WKWebView _processWillSwapOrDidExit]`.